lib/screens/login_screen/widgets/custom_login_widget.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:tsec_app/provider/theme_provider.dart';
+import 'package:tsec_app/utils/themes.dart';
+
+class LoginWidget extends StatefulWidget {
+  @override
+  _LoginWidgetState createState() => _LoginWidgetState();
+}
+
+class _LoginWidgetState extends State<LoginWidget> {
+  @override
+  Widget build(BuildContext context) {
+    var brightness = Theme.of(context).brightness;
+    bool isItDarkMode = brightness == Brightness.dark;
+    return Wrap(
+      children: [
+        Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 25, vertical: 20),
+            child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text("Email", style: Theme.of(context).textTheme.subtitle1),
+                  Padding(
+                    padding: const EdgeInsets.only(top: 5, bottom: 10),
+                    child: DecoratedBox(
+                      decoration: BoxDecoration(
+                          borderRadius: BorderRadius.circular(18),
+                          boxShadow: isItDarkMode
+                              ? shadowLightModeTextFields
+                              : shadowDarkModeTextFields),
```

Comment:
```dart
boxShadow: isItDarkMode
                              ? shadowLightModeTextFields
                              : shadowDarkModeTextFields,
)
```

add a comma after `shadowDarkModeTextFields` for better formatting

Reply:
Done